src/test/java/org/folio/marc4ld/mapper/field043/Marc2Bibframe043IT.java

```diff
@@ -40,9 +40,12 @@ void whenMarcField043WithMultipleSubfield_a_map_shouldConvertOnlyTheFirstOne() {
       PredicateDictionary.GEOGRAPHIC_COVERAGE,
       List.of(ResourceTypeDictionary.PLACE),
       Map.of(
-        "http://bibfra.me/vocab/marc/geographicCoverage", "https://id.loc.gov/vocabulary/geographicAreas/n-us",
+        "http://bibfra.me/vocab/marc/geographicCoverage",
+        "[https://id.loc.gov/vocabulary/geographicAreas/n-us, "
```

Comment:
Hi @aleksei-pronichev, How difficult is to change this  map from `Map<String, String>` to `Map<String, List<String>>`? If this is not very hard, can we do it please? Else, I can open a tech debt ticket for this.

Reply:
I've added to this task